internal/cache/file/cache_handle_test.go

```diff
@@ -524,10 +526,10 @@ func (cht *cacheHandleTest) Test_Read_WithNilFileHandle() {
 
 	n, cacheHit, err := cht.cacheHandle.Read(context.Background(), cht.bucket, cht.object, offset, dst)
 
-	ExpectNe(nil, err)
-	ExpectEq(0, n)
-	ExpectFalse(cacheHit)
-	ExpectEq(util.InvalidFileHandleErrMsg, err.Error())
+	assert.NotEqual(cht.T(), nil, err)
```

Comment:
Sorry missed this.
nit: asset.NotNil()
Other places too

Reply:
Done